src/backend/services/UserService.js

```diff
@@ -0,0 +1,18 @@
+import crypto from "crypto";
+import * as UserRepository from "../repositories/UserRepository";
+
+class User {
```

Comment:
This is a valid way of writing the services, but given that the QuotationsService is written in the other way (exporting lots of functions), we should be consistent between services.